nautobot/core/ui/object_detail.py

```diff
@@ -785,6 +785,8 @@ def get_extra_context(self, context: Context):
         request = context["request"]
         if self.context_table_key:
             body_content_table = context.get(self.context_table_key)
+            body_content_table_class = type(body_content_table)
+            body_content_table_model = body_content_table_class.Meta.model
```

Comment:
Is this two-liner necessary or can we just do `body_content_table_model = body_content_table.Meta.model`?